qml/Panopticon/Welcome.qml

```diff
@@ -0,0 +1,175 @@
+import QtQuick 2.4
```

Comment:
Yeah, I thought about it but I don't want to put even more code into this huge PR.